### ISSUE TITLE
Credo TrailingWhiteSpace False Positive Bug

### ISSUE THREAD

**Issue by mateuszkwiatkowski** [rrrene/credo#1203]
Hello,

First, thank you for creating credo. I think I spotted a bug.

Credo reports a trailing whitespace warning on a line that contains only:
-  Proper indentation (2 spaces)
-  Escaped triple quotes (\"\"\")
-  A newline character
-  No trailing whitespace

Credo Version: 1.7.12

I created a repo with reproduction here: https://github.com/mateuszkwiatkowski/credo-bug-repro

**Comment by rrrene** [comment]
Thanks for reporting this 😀 Great reproduction 😎 It should now be fixed on `master`.

You can try this by setting the Credo dep to

```elixir
{:credo, github: "rrrene/credo"}
```

Please report back if your issue is solved! 👍

**Comment by mateuszkwiatkowski** [comment]
This is  fixed. Thanks!
